content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,99 @@
-   If your screen is different from the screenshots, make sure that the
-   ``iot_pairing`` module is installed.
+#. Input the :guilabel:`Pairing Code` on the :abbr:`IoT (Internet of Things)` module of the database
+   and click on the :guilabel:`Pair` button.
 
 WiFi Connection
 ===============
 
-#. Power on the IoT Box
-
-#. Copy the token
+#. Connect all the devices that have to be connected with cables to the
+   :abbr:`IoT (Internet of Things)` box (ethernet, usb devices, etc.).
 
-   .. image:: connect/connect05.png
+#. Power on the :abbr:`IoT (Internet of Things)` Box.
 
-#. Connect to the IoT Box WiFi Network (make sure there is no ethernet
-   cable plugged in your computer).
+#. Copy the :guilabel:`Token` from the :guilabel:`WiFi connection`section in
+   :menuselection:`IoT` module of the Odoo database.
 
-   .. image:: connect/connect06.png
+#. Connect to the :abbr:`IoT (Internet of Things)` Box WiFi Network (make sure there is no ethernet
+   cable plugged in the computer).
 
-#. You will be redirected to the IoT Box Homepage (if it doesn't work,
-   connect to the IP address of the box). Give a name to your IoT Box (not
-   required) and paste the token, then click on next.
+   .. image:: connect/connect-iot-wifi.png
+      :align: center
+      :alt: WiFi networks available on the computer.
 
-   .. image:: connect/connect07.png
+#. Upon connecting to the :abbr:`IoT (Internet of Things)` WiFi a browser will automatically
+   redirect to the :abbr:`IoT (Internet of Things)` Box Homepage. Give a name to the
+   :abbr:`IoT (Internet of Things)` Box, paste the previously copied token into the
+   :guilabel:`Server Token` field, and then click on :guilabel:`Next`.
 
-   .. tip::
-      If you are on Runbot, do not forget to add the -all or -base in the
-      token (e.g. this token
-      **http://375228-saas-11-5-iot-f3f920.runbot16.odoo.com\|4957098401**
-      should become
-      **http://375228-saas-11-5-iot-f3f920-all.runbot16.odoo.com\|4957098401**).
+   .. image:: connect/server-token.png
+      :align: center
+      :alt: Enter the server token into the IoT box.
 
-#. Choose the WiFi network you want to connect with (enter the password
-   if there is one) and click on Submit. Wait a few seconds before being
-   redirected to your database.
+#. Choose the WiFi network that the :abbr:`IoT (Internet of Things)` box will connect with (enter
+   the password if there is one) and click on :guilabel:`Connect`. Wait a few seconds before being
+   redirected to the database. The computer may need to be manually connected back to WiFi.
 
-   .. image:: connect/connect08.png
+   .. image:: connect/configure-wifi-network-iot.png
+      :align: center
+      :alt: Configuring the WiFi for the IoT box.
 
-You should now see the IoT Box.
+The :abbr:`IoT (Internet of Things)` box should appear in the :menuselection:`IoT` module of the
+Odoo database.
 
-.. image:: connect/connect09.png
+.. image:: connect/iot-box-connected.png
```

Comment:
![image](https://user-images.githubusercontent.com/44636244/224356279-6151a58b-7167-4526-ab14-368555256ea5.png)
Changing the name is not really relevant, we should emphasize that the token should be entered in the server rather than in the name part. If I'm not wrong, it won't work if you put the token there :/

Reply:
I've added a manual connect section on the Connect page. Great suggestion. Thanks.